src/client/swagger/unity.yml

```diff
@@ -23,174 +23,197 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/billing/date':
+  '/billing':
```

Comment:
I don't love this top level resource being _a_ billing, because that's...not a noun. I'm not a fan of slapping an `info` suffix on things either though, and I can't think of a better name for this data clump.

These fields make sense under `Account` i.e. account.balance, account.payment_method, etc. but then you've got drag all these extra fields around anywhere you want an account.

Reply:
That's a good point, but also can't think of a better name right now; will consider other options as we implement it.